assets/src/edit-story/app/prepublish/constants.js

```diff
@@ -465,35 +459,44 @@ export const MESSAGES = {
               MIN_STORY_CHARACTERS
             )}
           </li>
-        </DashedList>
+        </ul>
       ),
     },
   },
   MEDIA: {
     VIDEO_IMAGE_TOO_SMALL_ON_PAGE: {
       MAIN_TEXT: __('Increase video or image size on the page', 'web-stories'),
       HELPER_TEXT: (
-        <DashedList>
-          <li>
-            <a
-              href={VIDEO_DOCUMENTATION_URL}
-              rel="noreferrer"
-              target="_blank"
-              onClick={(evt) => trackClick(evt, 'click_pre_publish_video_docs')}
+        <ul>
+          <li>
+            <TranslateWithMarkup
+              mapping={{
+                a: (
+                  <Link
+                    size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
+                    rel="noreferrer"
```

Comment:
💭 may as well stick `noopener` here so that the new tab/window doesn't have access to the previous page
```suggestion
                    rel="noreferrer noopener"
```

Reply:
great idea!